presentations.yaml

```diff
@@ -1,3 +1,69 @@
+- name: What does Kubernetes Steering Steer?
+  description: |
+    The Kubernetes Steering Committee is tasked with decision-making and oversight of the non-technical aspects of the Kubernetes project. This session will be broken into two parts: The first half will be an overview of what the committee is, and what it isn’t. What it’s tasked with, its importance, what it has accomplished to date and its top priorities for the year. The latter half will be focused on answering questions from Kubernetes project constituents and the wider Cloud Native community at large. If you’re curious or have a question about how one of the largest Open Source projects is governed, how that impacts you, or how you can leverage our learnings in your cloud-native projects’ governance journeys, we encourage you to come stop by for a conversation! NOTE: Had to swap speakers at the last minute (covid).
+  video: https://youtu.be/d3xQw_JE6So
+  slides: https://speakerdeck.com/mrbobbytables/what-does-kubernetes-steering-steer
+  date: 2023-04-19
+  presenters:
+    - name: Robert Killen
+      github: mrbobbytables
+    - name: Nabarun Pal
+      github: palnabarun
+  event:
+    name: KubeCon EU 2023
+    url: https://kccnceu2023.sched.com/event/1HyUq?iframe=no
+  language: EN
+  projects:
+    - Kubernetes
+  tags:
+    - Project Maintainer
+    - Ambassador
+    - KubeCon
```

Comment:
Since we have a specific `event` meta field, should we avoid using tags to designate events as well?